### DISCUSSION TITLE
How would you run over 100GB of data on a model like LSTMFCNClassifier if you can't read all that into memory at once.

### DISCUSSION THREAD

**Discussion by TannerCypret** [alan-turing-institute/sktime#7270] [General]
I have a LSTMFCNClassifier model that's working well for me using about 5 gigabytes of time series data. I'm reading this data in all at once into memory and passing that into the fit method. Suppose I want to use 100GB of data now. I don't have enough RAM to support that. Is there a way to pass in a stream to the data so that the model reads it in as it needs it for training? SciKitLearn has a partial_fit() method for its MLPClassifier that can incrementally train the classifier, but LSTMFCNClassifier doesn't have that. Any advice on how I can pass in a large quantity of data into LSTMFCNClassifier without having to read it in all at once?

**Comment by jobs-git** [comment]
Its not possible to use PyTorch DataLoader?
